### ISSUE TITLE
Add request statistics

### ISSUE THREAD

**Issue by rurseekatze** [rurseekatze/node-tileserver#10]
Add an internal logging and statistics to count the number of tiles which was requested in the last minute, hour, day, etc. Also analyse e.g. referers to detect abuse (robots, mass downloads, ...).

It should be possible to access these stats with an API call which returns this data in JSON format.

**Comment by rurseekatze** [comment]
By running node-tileserver behind Apache as proxy, we can use Apache for logging: https://github.com/rurseekatze/OpenRailwayMap/pull/524.

Now there is a demand for a tool that analyzes the logs produced by Apache. The question is if such a tool should be part of this repository or instead put into a separate repository. I think it might be useful to use another repository because such a tile request log analyzer might be also interesting for users of other renderers.

I will close this issue after a separate repository was created.

**Comment by DerDakon** [comment]
Yes, this would be usable by every tileserver that produces apache-like logfiles.
